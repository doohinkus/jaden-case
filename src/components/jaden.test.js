import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallowToJson } from 'enzyme-to-json';

import Jaden from './jaden';
Enzyme.configure({ adapter: new Adapter() });

describe('Jaden', () => {
  it('should render the jaden', () => {
  const tree = shallow(
     <Jaden text="Jaden" warning="" />
   );
   expect(tree).toMatchSnapshot();
 });
});
