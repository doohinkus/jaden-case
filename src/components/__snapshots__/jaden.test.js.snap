// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jaden should render the jaden 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Jaden
    text="Jaden"
    warning=""
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p
          className="text-center"
>
          <img
                    alt="jaden smith"
                    className="img-fluid img-thumbnail clearbg"
                    src="jaden-cutout.png"
          />
</p>,
        <p
          className="box1 sb6"
          id="output"
>
          Jaden
          <span
                    style={
                              Object {
                                        "color": "red",
                                        "display": "block",
                                        "fontWeight": "bold",
                                      }
                    }
          />
</p>,
      ],
      "className": "jaden",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="jaden smith"
            className="img-fluid img-thumbnail clearbg"
            src="jaden-cutout.png"
/>,
          "className": "text-center",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "jaden smith",
            "className": "img-fluid img-thumbnail clearbg",
            "src": "jaden-cutout.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Jaden",
            <span
              style={
                            Object {
                                          "color": "red",
                                          "display": "block",
                                          "fontWeight": "bold",
                                        }
              }
/>,
          ],
          "className": "box1 sb6",
          "id": "output",
        },
        "ref": null,
        "rendered": Array [
          "Jaden",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "",
              "style": Object {
                "color": "red",
                "display": "block",
                "fontWeight": "bold",
              },
            },
            "ref": null,
            "rendered": "",
            "type": "span",
          },
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p
            className="text-center"
>
            <img
                        alt="jaden smith"
                        className="img-fluid img-thumbnail clearbg"
                        src="jaden-cutout.png"
            />
</p>,
          <p
            className="box1 sb6"
            id="output"
>
            Jaden
            <span
                        style={
                                    Object {
                                                "color": "red",
                                                "display": "block",
                                                "fontWeight": "bold",
                                              }
                        }
            />
</p>,
        ],
        "className": "jaden",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="jaden smith"
              className="img-fluid img-thumbnail clearbg"
              src="jaden-cutout.png"
/>,
            "className": "text-center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "jaden smith",
              "className": "img-fluid img-thumbnail clearbg",
              "src": "jaden-cutout.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Jaden",
              <span
                style={
                                Object {
                                                "color": "red",
                                                "display": "block",
                                                "fontWeight": "bold",
                                              }
                }
/>,
            ],
            "className": "box1 sb6",
            "id": "output",
          },
          "ref": null,
          "rendered": Array [
            "Jaden",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "",
                "style": Object {
                  "color": "red",
                  "display": "block",
                  "fontWeight": "bold",
                },
              },
              "ref": null,
              "rendered": "",
              "type": "span",
            },
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
